Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM camptocamp/qgis-server:latest
+
+RUN pip3 install --upgrade pip && \
+    pip3 install --no-cache-dir pytest
+# used for calling the qgis api for the tests
+RUN cp -r /usr/local/share/qgis/python/qgis /usr/local/lib/python3.6/dist-packages/
```

Comment:
wouldn't a symlink be sufficient instead of copying ?

Reply:
Better add it to PYTHONPATH